CSSTooltipPseudo/explainer.md

```diff
@@ -0,0 +1,896 @@
+tootlips can escape the bounds of the browser window today (see
+[Security Considerations](#security-considerations) for more details).
+
+Given the proposal to not allow tooltips styled through the
+`::tooltip` psuedo element to escape the bounds of the windows (see
+[Security Considerations](#security-considerations) for more details),
+this proposal also includes the ability for authors to customize the
+tooltip positioning.
+
+As such, this proposal will utilize [CSS Anchor Positioning](
+https://drafts.csswg.org/css-anchor-position-1/) to define the default
+position of the `::tooltip` pseudo element, which will allow authors
+to override this in the way that they see fit.
+
+As a result, this mean that the element associated with the `title`
+attribute (or `<title>` element in the case of SVG) will be an implied
+anchor for the rendered tooltip.
+
+### Default styles for `::tooltip`
+
+When the new default `::tooltip` styles are tiggered via the
+`appearance` property, a set of default styles would be applied
+by the UA.
+
+These styles were mostly pulled directly from Lea Verou's initial
+[proposal](
+https://github.com/w3c/csswg-drafts/issues/8930) with a couple of
+minor edits based on the new proposal for an updated definition of
+`content: normal` within the `::tooltip` pseudo element, as well
+as the proposal to define the position via CSS Anchor Positioning
+(which was included as an option in Lea's original proposal, as
+well).
+
+```css
+/* UA styles */
+::tooltip {
+  /* content: normal; is implied */
+  color: InfoText;
+  background: InfoBackground;
+  font-size: .8rem;
+  box-shadow: .1rem .1rem .2rem rgb(0 0 0 / .2);
+  position: absolute;
+  position-area: top right;
+  position-try-fallbacks: top left, top center, bottom right, bottom left, bottom center, center right, center left;
+  transition: 0s 3s visibility;
+  
+  @starting-style {
+    visibility: hidden;
+  }
+}
+```
+
+These proposed set of initial UA styles is a starting point and is
+open to further discussion.
+
+To see some of these styles in action, check out the [CodePen demo](
+https://codepen.io/alisonmaher/pen/MYYxpJZ) utilizing `interesttarget`
+to render the tooltip. You'll note that this requires some changes
+to the styles above (notably, this doesn't include the defined
+`transition` and sets `margin` to `0` to account for the default
+`popover` styles). Since this is based on `interesttarget`, at the
+time of writing, this demo is required to be tested in a Chromium-based
+browser with Web Platform Experimental Flags Enabled.
+
+### What styles can be applied to `::tooltip`?
+
+Not all styles should or need to be applied within the context
+of a simple text-based tooltip. For example, we likely don't
+want tooltips with images, lists, scrollbars, or non-default
+`display` types or `z-index`. As such, there should be a limited list of styles
+that are valid within a `::tooltip` pseudo element; that list of
+which can be expanded on in the future.
```

Comment:
"that list of which can be expanded on in the future." -> "that list can be expanded on in the future."

Reply:
Done